Lib/pydoc.py

```diff
@@ -2059,9 +2059,10 @@ def interact(self):
         while True:
             try:
                 request = self.getline('help> ')
-                if not request: break
             except (KeyboardInterrupt, EOFError):
                 break
+            if not request or request.isspace():
+                continue  # back to the prompt
             request = request.strip()
 
```

Comment:
I still think the following is cleaner than having the extra `.isspace` call.  I don't think we're actually saving a string allocation that way, either, since in the case where we would be continuing, `.strip()` would take us down to the empty string, which is interned.

But this is a judgement call, and it's ultimately up to @AA-Turner.

```suggestion
            request = request.strip()
            if not request:
                continue  # back to the prompt
```